staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_tests.go

```diff
@@ -0,0 +1,742 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcd3
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math"
+	"reflect"
+	"strconv"
+	"sync"
+	"testing"
```

Comment:
anything that imports `testing` either needs to be in a `_test.go` file or in a dedicated `testing` subpackage... putting this here means that the `testing` package gets linked into production binaries

Reply:
ACK - @wojtek-t looks like there's no way around a new package for this.

Reply:
OK - we'll figure out the 0% coverage aspect of that test later.